samples/bluetooth/peripheral_lbs/README.rst

```diff
@@ -85,18 +112,18 @@ This testing procedure assumes that you are using `nRF Connect for Mobile`_.
 #. Connect to the device from nRF Connect (the device is advertising as "Nordic_LBS").
 #. Observe that the services of the connected device are shown.
 #. In "Nordic LED Button Service", click the :guilabel:`Play` button for the "Button" characteristic.
-#. Press **Button 1** either on the dongle or on the development kit.
+#. Press **Button 1** on the device.
 #. Observe that notifications with the following values are received:
 
    * ``00`` when **Button 1** is released,
    * ``01`` when **Button 1** is pressed.
 
-#. Control the color of RGB LED on the dongle or status of **LED 3** on the kit by writing the following values to the "LED" characteristic in the "Nordic LED Button Service":
+#. Control the color of RGB LED on the dongle/Thingy:53 or status of **LED 3** on the kit by writing the following values to the "LED" characteristic in the "Nordic LED Button Service":
 
-   * ``00`` to switch the LED off on the kit or turn on the red RGB LED on the dongle.
-   * ``01`` to switch the LED on on the kit or turn on the green RGB LED on the dongle.
+   * ``00`` to switch the LED off on the kit or turn off the green RGB LED on the dongle or turn off the blue RGB LED on Thingy:53.
```

Comment:
maybe, instead of overwriting, we should have two variants: one for dongle and another for thingy

Reply:
Or maybe we could even call it e.g. "user controlled LED" and explain which hardware LED is used on which board in `User interface` subsection.?

Reply:
@greg-fer what's your opinion on this?